python/nx-cugraph/nx_cugraph/convert.py

```diff
@@ -254,7 +265,23 @@ def from_networkx(
         key_to_id = None
     else:
         col_iter = map(key_to_id.__getitem__, col_iter)
-    col_indices = cp.fromiter(col_iter, np.int32)
+    if graph.is_multigraph():
+        col_indices = np.fromiter(col_iter, np.int32)
+        num_multiedges = np.fromiter(
+            map(len, concat(map(dict.values, adj.values()))), np.int32
+        )
+        # cp.repeat is slow to use here, so use numpy instead
```

Comment:
I didn't catch this before but we should probably add a dependency on `numpy` for this (instead of assuming we'll get it transitively through `cupy` or something else)

Reply:
Will do!

Reply:
Done.